src/auth-modal-component/AuthModal.module.css

```diff
@@ -0,0 +1,42 @@
+:root {
+    --primary-color: rgb(102, 51, 153);
+    --secondary-color: rgb(147, 197, 253);
+    --white: rgb(255, 255, 255);
+    --grey: rgb(175, 168, 168);
+    --light-grey: #ebebeb;
```

Comment:
use RGB or just Hex code

Reply:
thanks for review, will do these changes